standalone_tests/artifact_object_reference_test.py

```diff
@@ -708,27 +708,66 @@ def test_image_reference_with_preferred_path():
     
     # This test just checks that all this logic does not fail
 
+def test_distributed_artifact_simple():
+    table_name = "dataset"
+    artifact_name = "simple_dist_dataset"
+    group_name = "test_group_{}".format(np.random.rand())
+    artifact_type = "dataset"
+    columns = ["A", "B", "C"]
+    count = 2
+    images = []
+    image_paths = []
+
+    # Add Data
+    for i in range(count):
+        run = wandb.init(project=WANDB_PROJECT, group=group_name)
+        artifact = wandb.Artifact(artifact_name, type=artifact_type)
+        image = wandb.Image(np.random.randint(0, 255, (10, 10)))
+        path = "image_{}".format(i)
+        images.append(image)
+        image_paths.append(path)
+        artifact.add(image, path)
+        run.upsert_artifact(artifact)
+        run.finish()
+
+    # TODO: Should we try to use_artifact in some way before it is finished?
+
+    # Finish
+    run = wandb.init(project=WANDB_PROJECT, group=group_name)
+    artifact = wandb.Artifact(artifact_name, type=artifact_type)
+    # artifact.add_file("./test.py")
+    run.finish_artifact(artifact)
+    run.finish()
+
+    # test
+    run = wandb.init(project=WANDB_PROJECT)
+    artifact = run.use_artifact("{}:latest".format(artifact_name))
+    assert len(artifact.manifest.entries.keys()) == count * 2
+    # for image, path in zip(images, image_paths):
+    #     assert image == artifact.get(path)
+
 if __name__ == "__main__":
     _cleanup()
     test_fns = [
-        test_artifact_add_reference_via_url,
-        test_add_reference_via_artifact_entry,
-        test_adding_artifact_by_object,
-        test_get_artifact_obj_by_name,
-        test_image_reference_artifact,
-        test_nested_reference_artifact,
-        test_table_slice_reference_artifact,
-        test_image_refs,
-        test_point_cloud_refs,
-        test_bokeh_refs,
-        test_html_refs,
-        test_video_refs,
-        test_table_refs,
-        test_joined_table_refs,
-        test_audio_refs,
-        test_joined_table_referential,
-        test_joined_table_add_by_path,
-        test_image_reference_with_preferred_path,
+        # test_artifact_add_reference_via_url,
```

Comment:
@tssweeney Do we want to uncomment these before merging?

Reply:
yes, and we should run the regression. this was a leftover.